### ISSUE TITLE
Broken link

### ISSUE THREAD

**Issue by mihaisavezi** [ing-bank/lion#76]
The last link in README.md to forms is broken.

--> https://github.com/ing-bank/lion/blob/master/docs/forms.md

I assume the correct one is this --> https://github.com/ing-bank/lion/blob/master/packages/field/docs/FormFundaments.md

**Comment by bashmish** [comment]
@tlouisse did you move general form documentation? I thought we wanted to keep it in `/root/docs`?

**Comment by tlouisse** [comment]
Didn't move anything... ;)

Think this link should actually be removed, the doc for it didn't exist (or maybe it was an empty file and I removed, could be).

The FieldCustomMixin never became what it originally was intended to be: https://github.com/ing-bank/lion/issues/28
So this mixin makes only sense on our extension layer of Lion; different repo.

Currently the term 'custom field' is already 'occupied' and this concept should therefore be renamed to 'SpecializedField'.


**Long story short: the broken link should be removed :)**
